spec/system/task_spec.rb

```diff
@@ -79,7 +81,7 @@
         visit tasks_path
         click_on '終了期限でソートする'
         sleep 2
```

Comment:
必要ない`sleep`はテスト実行時の時間的コストがかかるので削除した方がいいですね。
必要なものは下記のようにコメントがあると意図が伝わります。
```suggestion
        sleep 2 # テスト失敗を回避
```

Reply:
こちらで修正
110a1b8ef01e75f5478e608f894d3e87b11dd449

Reply:
確認ですがこのsleepはないとテストは失敗しますか？
また、3秒もsleepさせる必要はありますか？

Reply:
sleepがないとテストは失敗になります。本日メンターに質問をしてベストではないが、sleepを入れて置くようになりました。　秒数はあてです。

Reply:
> 秒数はあてです。

毎回3秒かかるテストケースがいくつもあると、積み重ねでテストコード実行時の時間的コストが積み上がっていきます。
必要なければ sleepの待機時間は 0.5 または 1 に変更してください。2以上は長過ぎます。